modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -74,12 +74,18 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
             }
             {{/isKeyInQuery}}
             {{/isApiKey}}
-            {{#isBasic}}
+            {{#isBasicBasic}}
             // http basic authentication required
             if (configuration && (configuration.username || configuration.password)) {
                 localVarHeaderParameter["Authorization"] = "Basic " + btoa(configuration.username + ":" + configuration.password);
             }
-            {{/isBasic}}
+            {{/isBasicBasic}}
+            {{#isBasicBearer}}
+            // http bearer authentication required
+            if (configuration && configuration.password) {
+                localVarHeaderParameter["Authorization"] = "Bearer " + btoa(configuration.password);
```

Comment:
wouldn't it be better to use the `configuration.accessToken`?
https://github.com/OpenAPITools/openapi-generator/blob/0889b8ec1349d6f912ad7dfa221d3980f1801dba/modules/openapi-generator/src/main/resources/typescript-axios/configuration.mustache#L34-L40

this is also used e.g. in the angular client:
https://github.com/OpenAPITools/openapi-generator/blob/0889b8ec1349d6f912ad7dfa221d3980f1801dba/modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache#L213-L220

Reply:
You're right, will fix

Reply:
fixed in https://github.com/OpenAPITools/openapi-generator/pull/2578/commits/dbc340061bd57fd2bb5f6442d7421d2ded561653

thanks